doc/packet_trimming/packet-trimming-design.md

```diff
@@ -301,39 +347,43 @@ The new action will be handled as part of existing `PACKET_ACTION`.
 
 In order to use trimming configuration in ACL rule, a dedicated ACL table type must be defined.
 
-### 2.3.5 Flexcounter orch
+### 2.5.6 Flexcounter orch
 
 Existing infrastructure will be reused.
 
 Flex counter groups `port/queue` will be extended with a new SAI attributes  
 `SAI_PORT_STAT_TRIM_PACKETS/SAI_QUEUE_STAT_TRIM_PACKETS` respectively.
 
-## 2.4 DB schema
+## 2.6 DB schema
 
-### 2.4.1 Config DB
+### 2.6.1 Config DB
 
-#### 2.4.1.1 Switch OA
+#### 2.6.1.1 Switch OA
 ```abnf
 ; defines schema for switch trimming configuration attributes
 key = SWITCH_TRIMMING|GLOBAL ; switch trimming global. Must be unique
 
-; field           = value
-size              = 1*10DIGIT   ; size (in bytes) to trim eligible packet
-dscp_value        = 1*2DIGIT    ; DSCP value assigned to a packet after trimming
-queue_index       = queue-index ; queue index to use for transmission of a packet after trimming
+; field     = value
+size        = 1*10DIGIT   ; size (in bytes) to trim eligible packet
+dscp_value  = dscp-value  ; DSCP value assigned to a packet after trimming
+tc_value    = 1*3DIGIT    ; TC value assigned to a packet after trimming
```

Comment:
@nazariig why 3DIGIT for TC value? Should we restrict to 2DIGIT?

Reply:
@kperumalbfn SAI spec says:
```
    /**
     * @brief New packet trim TC value
     *
     * @type sai_uint8_t
     * @flags CREATE_AND_SET
     * @default 0
     * @validonly SAI_SWITCH_ATTR_PACKET_TRIM_DSCP_RESOLUTION_MODE == SAI_PACKET_TRIM_DSCP_RESOLUTION_MODE_FROM_TC
     */
    SAI_SWITCH_ATTR_PACKET_TRIM_TC_VALUE,
```

Type `uint8_t` -> 8 bit -> max 255 aka 3 digits